lib/elixir_internal_certificate_web/views/user_settings_view.ex

```diff
@@ -0,0 +1,3 @@
+defmodule ElixirInternalCertificateWeb.UserSettingsView do
```

Comment:
should we delete this

Reply:
I saw this conversation was resolved without any reply and the file is still existing,  might I know the reason @nvminhtue was I wrong 🤔